### PR TITLE
docs: Clean up Moderator Handbook

### PR DESCRIPTION
**Pull request by SethFalco**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

I was just reading the Moderator Handbook and wanted to suggest a few changes.

* Make usage of sentence case or title case on headings more consistent. - [Wikipedia](https://en.wikipedia.org/wiki/Letter_case#Case_styles)
* Some sentences didn't explicitly refer to a noun, so connected them to the sentence before or added the noun. - [Wikipedia](https://en.wikipedia.org/wiki/Sentence_(linguistics)#Clauses)
* GitPod is written like Gitpod. - [Forum](https://community.gitpod.io/), [GitHub](https://github.com/gitpod-io/gitpod), [Gitpod](https://www.gitpod.io/), [Twitter](https://twitter.com/gitpod).
* Removed some redundant words or phrases.
* Codepen is written like CodePen. - [CodePen](https://codepen.io/), [YouTube](https://www.youtube.com/channel/UCZq5KpWOrPQ2kSm5C4Rmbrg), [Twitter](https://twitter.com/CodePen), [Instagram](https://www.instagram.com/codepen/).
* "username" is one word. - [Wordnik](https://www.merriam-webster.com/dictionary/username), [Merriam-Webster](https://www.merriam-webster.com/dictionary/username), [Collins](https://www.collinsdictionary.com/dictionary/english/username)
* Added commas to split clauses in PR templates.
* Other typos like duplicate words, misuse of words, wrong capitalization, or misspellings.

--- 

Two problems I left:
1. Not all headings were updated, since I wasn't sure which should be sentence case or title case.
1. "**We proposals like this all the time and is in the best position...**" - I have no idea what this is supposed to be.

First one doesn't matter, but I'd appreciate a comment on the second.

Also ran over it with [LanguageTool](https://languagetool.org/), so grammatically I think it should be sound. Some suggestions are subjective, so don't hesitate to disagree with anything.

### CODE REVIEW

File: docs/moderator-handbook.md
Comment (huyenltnguyen) [inline docs/moderator-handbook.md:331]: I _think_ this is supposed to be

> We get proposals like this all the time and the staff members are in the best position to judge whether such a relationship will be worth it for our community (and it rarely is).

But it definitely needs confirmation from the core team.

**Review by huyenltnguyen — COMMENTED** [review]
Thank you for your attention to detail, and I LOVE LOVE LOVE the intensive research to get the words right ❤️ 

I don't have opinions on the casing of the headings, I personally think they can go either way. One thing I noticed in the doc though, is the inconsistency in the use of "pull request" and "pull-request". I think we can change it to "pull request" while we are at it, but your call 😄

**Comment by SethFalco** [comment]
> "pull request" and "pull-request"

Didn't notice that at all, thank you for pointing it out!

I'll wait for confirmation before pushing your suggestion; it definitely makes much more sense.

File: docs/moderator-handbook.md
Comment (raisedadead) [reply docs/moderator-handbook.md:331]: @huyenltnguyen is correct. It is indeed a typo.

Comment (raisedadead) [reply docs/moderator-handbook.md:331]:
```suggestion
We get proposals like this all the time and is in the best position to judge whether such a relationship will be worth it for our community (and it rarely is).
```

**Review by raisedadead — CHANGES_REQUESTED** [review]

**Review by raisedadead — APPROVED** [review]

**Pull request merged**
